README.md

```diff
@@ -125,3 +127,47 @@ VAPID keys can be generated using the following command:
 ```bash
 npx web-push generate-vapid-keys
 ```
+
+To set up Keycloak, follow these steps
+- Run it with `docker compose up -d keycloak`
+- Go to `http://localhost:8080/auth`.
+- Create a new realm called `apollusia`
+- Create a client called `web` with the following options:
+  - Valid Redirect URLs: `http://localhost:4200/*`
+  - Valid Post Logout Redirect URLs: `+`
+  - Web Origins: `+`.
+- Under "Realm Settings > Login", configure:
+  - User registration: On
+  - Forgot password: On
+  - Remember me: On
+  - Email as username: On
+  - Login with email: On
+  - Duplicate emails: Off
+  - Verify email: Off
+- Under "Realm Settings > User profile", create an attribute:
+  - Attribute [Name]: pushTokens
+  - Display Name: Push Tokens
+  - Multivalued: On
+  - Attribute Group: user-metadata
+  - Who can edit?: User, Admin
+  - Who can view?: User, Admin
+- Create a user as follows:
+  - Username/email: admin@apollusia.com
+  - First Name: Apollusia
+  - Last Name: Admin
+  - Email Verified: On
+  - Credentials > Password: `root`
+  - Role Mapping > Assign Role > Filter by clients > Select all (the list may be long, change pagination to 100 elements to see all) > Assign
+  - Role Mapping > Assign Role > Filter by realm roles > Select all > Assign
```

Comment:
```suggestion
- Create a user as follows:
  - Email Verified: Yes
  - Username/email: admin@apollusia.com
  - First Name: Apollusia
  - Last Name: Admin
  - Hit Create
  - Credentials > Set Password: `root` 
  - Role Mapping > Assign Role > Filter by clients > Select all (the list may be long, change pagination to 100 elements to see all) > Assign
  - Role Mapping > Assign Role > Filter by realm roles > Select all > Assign
```

Reply:
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user